modules/nw-kuryr-migration.adoc

```diff
@@ -297,9 +315,9 @@ $ oc get co
 +
 The status of every cluster Operator must be the following: `AVAILABLE="True"`, `PROGRESSING="False"`, `DEGRADED="False"`. If a cluster Operator is not available or degraded, check the logs for the cluster Operator for more information.
 
-. Complete the following steps only if the migration succeeds and your cluster is in a good state:
+IMPORTANT: Do not proceed if the any of the previous verification steps indicate errors.
 
-.. To remove the migration configuration from the CNO configuration object, enter the following command:
+. If the migration completed and your cluster is in a good state, remove the migration configuration from the CNO configuration object by entering the following command:
```

Comment:
> If the migration is completed and your cluster is in a good state

Should we define what is a cluster in a good state? As far as I understand, at this point, it is expected to have some pods (multus pods, for example) stuck in the Terminating state until the Kuryr finalizers are removed in the next steps.

Reply:
@imatza-rh we can add this as subsequent patch for the final version.

Reply:
@gryf Any changes incoming with respect to this, too?

Reply:
ditto.

Reply:
@maxwelldb added a note about this.